README.md

```diff
@@ -1 +1,66 @@
-# gatk4-CNNVariant
\ No newline at end of file
+# gatk4-cnn-variant-filter
+
+### Purpose :
+This repo provides workflows that takes advangate of GATKs CNN tool which a deep learning 
+approach to filter variants based on Convolutional Neural Networks. 
+
+Please read the following discussion to learn more about the CNN tool: [Deep Learning in GATK4](https://gatkforums.broadinstitute.org/gatk/discussion/10996/deep-learning-in-gatk4).
+
+### cram2filtered.wdl
+This workflow takes an input CRAM/BAM to call variants with HaplotypeCaller
+then filters the calls with the CNNVariant neural net tool using the filtering model specified.
+
+The site-level scores are added to the `INFO` field of the VCF. The architecture arguments,
+`info_key` and `tensor_type` arguments MUST be in agreement (e.g. 2D models must have
+`tensor_type` of `read_tensor` and `info_key` of `CNN_2D`, 1D models have `tensor_type` of
+`reference` and `info_key` of `CNN_1D`). The `INFO` field key will be `1D_CNN` or `2D_CNN`
+depending on the neural net architecture used for inference. The architecture arguments
+specify pre-trained networks. New networks can be trained by the GATK tools: CNNVariantWriteTensors 
+and CNNVariantTrain. The CRAM could be generated by the [single-sample pipeline](https://github.com/gatk-workflows/gatk4-data-processing/blob/master/processing-for-variant-discovery-gatk4.wdl).
+If you would like test the workflow on a more representative example file, use the following 
+CRAM file as input and change the scatter count from 4 to 200: gs://gatk-best-practices/cnn-h38/NA12878_NA12878_IntraRun_1_SM-G947Y_v1.cram.
+
+#### Requirements/expectations :
+ - CRAM/BAM
+ - BAM Index (if input is BAM) 
+
+#### Output :
+ - Filtered VCF 
+ - Filtered VCF index. 
+
+### cram2model.wdl
+Creates a filtering model to be used in the cram2filtered workflow.
```

Comment:
It's worth mentioning that this is optional.  Grad students don't always read the instructions with a fine toothed comb.